docs/topics/installing-mta-disconnected-environment.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * docs/
+
+:_content-type: PROCEDURE
+[id="installing-mta-disconnected-environment_{context}"]
+= Installing {ProductShortName} on a disconnected environment
+
+On a connected device, first download and save the binary. Then download and save the podman images required, you require the {ProductShortName} podman image and the respective provider image, you require.
+
+* Download the respective binary from the link:https://developers.redhat.com/products/mta/download[customer portal]:
+** CLI for Linux aarch64
+** CLI for MacOS x86_64
+** CLI for MacOS aarch64
+** CLI for Windows x86_64
+** CLI for Windows aarch64
+
+* On a connected device, download and save the images.
+
+* Copy the binary to the disconnected device.
+
+In addition, they need to save and import the associated container images.
+
+== Downloading the Podman images
+
+.Prerequisites
+
+* Podman is installed. For more information, see
+link:https://podman.io/[Podman].
+
+.Procedure
+
+. Use Podman to authenticate to
+link:registry.redhat.io[registry.redhat.io]:
+
+[source,terminal]
+----
+$ podman login registry.redhat.io
+----
+
+. Enter your username and then your password for registry.redhat.io:
+
+[source,terminal]
+----
+Username: <registry-service-account-username>
+Password: <registry-service-account-password>
+----
+
+* You should see the following output:
+
+[source,terminal]
+----
+Login Succeeded!
+----
+
+. Use podman to pull the image from the registry:
+
+[source,terminal]
+----
+$ podman pull registry.redhat.io/mta/mta-rhel9-operator:7.1.0
+----
+
+. Use podman to pull the respective provider image that you need from the
+registry:
+
+.. For Java, run: +
+
+[source,terminal]
+----
+$ podman pull registry.redhat.io/mta/mta-java-external-provider-rhel9:7.1.0
+----
+
+.. For .NET, run: +
+
+[source,terminal]
+----
+$ podman pull registry.redhat.io/mta/mta-dotnet-external-provider-rhel9
+----
+
+. Save the images:
+
+[source,terminal]
+----
+$ podman save <image> -o <my_image.image>
+----
+
+. Copy the `+.image+` file onto a USB or directly to the file system of
```

Comment:
Please mention that binary should be copied as well. Too many users that are doing ONLY what is written.